apps/CEventService.API/Services/IService.cs

```diff
@@ -0,0 +1,9 @@
+namespace CEventService.API.Services
+{
+    public interface IService<T,T_DTO>
```

Comment:
The same as with the IRepository happens here, methods with names very committed only to services.

Reply:
Same answer as in the Irepository here..
> Let me understand well, you are asking to erase the word asyn aftert each method name? Also the skip and take parameters are well documented in its official docs. Do you want me to copy them as a comment in the file? @CatCoder36

Reply:
Same response